README.md

```diff
@@ -435,11 +435,59 @@ and
 
 ### Deploying on Docker
 
-You can run the service that we developed above as a Docker container. As the Ballerina platform includes a [Ballerina_Docker_Extension](https://github.com/ballerinax/docker), which offers native support for running ballerina programs on containers, you just need to put the corresponding Docker annotations on your service code. 
+You can run the service that we developed above as a Docker container. As the Ballerina platform includes a [Ballerina_Docker_Extension](https://github.com/ballerinax/docker), which offers native support for running ballerina programs on containers, you just need to put the corresponding Docker annotations on your service code.
```

Comment:
We cannot say "Ballerina platform" here since we are changing our approach. Let's go with something like "As Ballerina includes...".

Please change all mentions of "ballerina" to "Ballerina". Apply elsewhere if applicable.